### DISCUSSION TITLE
How many connection & How to increased?

### DISCUSSION THREAD

**Discussion by SpiderChat** [owncast/owncast#1364] [Q&A]
Dear

How many people can watching our viedo?
How to increased to more then 100 people watching?

_Sorry for my broken english_

**Comment by gabek** [comment]
There is no limit to the amount of viewers, but every server/network/environment is different.  Generally the easiest answer to scale up your viewership is to use an external storage provider.

Read more details about how to increase your viewership here: https://owncast.online/docs/scaling/

**Reply by SpiderChat** [reply]
Thanks
